reana_job_controller/job_monitor.py

```diff
@@ -216,6 +219,13 @@ def get_job_logs(self, job_pod):
                         container=container.name,
                     )
                     pod_logs += "{}: :\n {}\n".format(container.name, container_log)
+                    if hasattr(container.state.terminated, "reason"):
```

Comment:
Note that this is not filtering the reasons, and because `Completed` is a valid reason, now finished jobs would have as last log line `Completed`, e.g.:
```console
$ rc logs -w workflow
==> Workflow engine logs
2021-03-23 09:32:20,563 | root | MainThread | INFO | Publishing step:0, cmd: python "code/helloworld.py" --inputfile "data/names.txt" --outputfile "results/greetings.txt" --sleeptime 0, total steps 1 to MQ
2021-03-23 09:32:26,590 | root | MainThread | INFO | Workflow 6b8978c8-dcf9-4348-8365-6947126e1844 finished. Files available at users/00000000-0000-0000-0000-000000000000/workflows/6b8978c8-dcf9-4348-8365-6947126e1844.




==> Job logs
==> Step: ec2fc8de-d665-424d-bdb2-e440dd9cbb6f
==> Workflow ID: 6b8978c8-dcf9-4348-8365-6947126e1844
==> Compute backend: Kubernetes
==> Job ID: reana-run-job-3a03950d-24e9-4188-ae46-1ef7ed8c5728
==> Docker image: python:2.7-slim
==> Command: python "code/helloworld.py" --inputfile "data/names.txt" --outputfile "results/greetings.txt" --sleeptime 0
==> Status: finished
==> Logs:
job: :
 2021-03-23T09:32:21 INFO     Parameters: inputfile=data/names.txt outputfile=results/greetings.txt sleeptime=0.0


Completed
```
This can be easily amended, let me know what you think.

Reply:
I think it is OK to show the real status, I would even keep `OOMKilled` and not translate it... There is a certain advantage of keeping final message exactly as it was, and users using REANA locally with Kind might like that as well.

Also, showing `Completed` is fully fine, it is actually a nice stamp that everything went fine.

Suggestion: always job output reason, and do not even do `8S_STATE_REASON_TO_HUMAN_READABLE` translation business?